pkg/operatorsource/handler_test.go

```diff
@@ -22,7 +23,10 @@ func TestHandle_PhaseHasChanged_UpdateExpected(t *testing.T) {
 	controller := gomock.NewController(t)
 	defer controller.Finish()
 
-	kubeclient := mocks.NewKubeClient(controller)
+	// Making two OperatorSource objects that are not equal to simulate a change.
+	opsrcIn, opsrcOut := helperNewOperatorSourceWithEndpoint("marketplace", "foo", "remote"), helperNewOperatorSourceWithEndpoint("marketplace", "foo", "local")
```

Comment:
Why are you moving this code?

Reply:
@aravindhp we need `opsrcIn` to set up the fake client with an OperatorSource. The mock client did not need the opsrcIn, hence the `opsrcIn` was initialized after the kubeclient. Since I need the `OpsrcIn` for the fakeclient, I had to move it up to initialized it first and feed it to the fakeclient.